src/core/server/integration_tests/saved_objects/migrations/group2/check_target_mappings.test.ts

```diff
@@ -122,7 +122,7 @@ describe('migration v2 - CHECK_TARGET_MAPPINGS', () => {
 
       // Check for migration steps present in the logs
       logs = await fs.readFile(logFilePath, 'utf-8');
-      expect(logs).not.toMatch('CREATE_NEW_TARGET');
+      expect(logs).not.toMatch('[.kibana] CREATE_NEW_TARGET');
```

Comment:
The test now creates a new index for the new type.
I made the constraint more specific and ensure that a new .kibana is not being created.